appinventor/components/src/com/google/appinventor/components/runtime/ChartDataBase.java

```diff
@@ -50,6 +52,8 @@
 
     private boolean initialized = false; // Keep track whether the Screen has already been initialized
 
+    private int t = 1;
```

Comment:
Support for on the fly data sensor adding to sync t (?)

Reply:
Also might be worthwhile making the t value part of a class which covers all the Data Models (Chart component/Chart View)

Reply:
Covered in #122